docs/OnlineTutorial/guide.md

```diff
@@ -1472,8 +1472,11 @@ So far we have only considered a handful of values at a time in local
 variables. Now we turn our attention to arrays of data. Arrays are a built-in
 part of the language, with their own type, `array<T>`,
 where `T` is another type. For now we only consider
-arrays of integers, `array<int>`. Arrays can be `null`, and have a
-built-in length field, `a.Length`.
+arrays of integers, with type `array<int>`;
```

Comment:
This specialisation should probably move after the use of the type variable in the following lines.

Reply:
Done